From bcd942b76a0e286d4260cf00bccbdbfd8e8025ee Mon Sep 17 00:00:00 2001
From: Ilies CHERGUI <ichergui@nvidia.com>
Date: Wed, 20 Aug 2025 17:43:13 +0100
Subject: [PATCH 08/10] Skip model download for object_detection_torch

This step requires a native installation of PyTorch, which
is not available in the OE build. If this model is needed
then it needs to be manually downloaded and converted
outside of the OE build process.

Upstream-Status: Inappropriate [OE-specific]
Signed-off-by: Ian Stewart <istewart@nvidia.com>
---
 applications/object_detection_torch/CMakeLists.txt | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/applications/object_detection_torch/CMakeLists.txt b/applications/object_detection_torch/CMakeLists.txt
index 24529297..6f00034d 100644
--- a/applications/object_detection_torch/CMakeLists.txt
+++ b/applications/object_detection_torch/CMakeLists.txt
@@ -52,14 +52,6 @@ if(HOLOHUB_DOWNLOAD_DATASETS)
     GXF_ENTITIES_FRAMERATE 30
   )
   add_dependencies(object_detection_torch object_detection_torch_data)
-
-  # Download the model
-  add_custom_target(download_model
-    COMMAND python3 "${CMAKE_CURRENT_SOURCE_DIR}/generate_resnet_model.py"  "${HOLOHUB_DATA_DIR}/object_detection_torch/frcnn_resnet50_t.pt"
-    DEPENDS object_detection_torch_data
-    BYPRODUCTS "${HOLOHUB_DATA_DIR}/object_detection_torch/frcnn_resnet50_t.pt"
-  )
-  add_dependencies(object_detection_torch download_model)
 endif()
 
 # Copy config file
-- 
2.34.1

